src/summary/documentation/dimensions-card.resource.ts

```diff
@@ -0,0 +1,51 @@
+import { openmrsObservableFetch } from "@openmrs/esm-api";
+import { forkJoin } from "rxjs";
+import { map } from "rxjs/operators";
+import { Dimensions } from "./dimensions";
+
+const HEIGHT_CONCEPT = "5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const WEIGHT_CONCEPT = "5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+export function getDimensions(patientId: string) {
+  return forkJoin({
+    weights: getDimensionObservations(patientId, "weight"),
+    heights: getDimensionObservations(patientId, "height")
+  }).pipe(map(data => formatDimensions(data.weights, data.heights)));
+}
+
+function getDimensionObservations(
+  patientId: string,
+  type: "weight" | "height"
+) {
+  const concept = type === "weight" ? WEIGHT_CONCEPT : HEIGHT_CONCEPT;
+  return openmrsObservableFetch(`/ws/fhir/Observation?subject:
+                Patient=${patientId}&code=${concept}`).pipe(
+    map(({ data }) => data["entry"]),
```

Comment:
Out of curiosity, why not:

```suggestion
    map(({ data }) => data.entry),
```

Typescript reasons?

Reply:
yes typescript doesn`t like `data.entry` for some reason.